packages/actions/src/helpers/utils.ts

```diff
@@ -0,0 +1,115 @@
+import fs from "fs"
+import { CircuitMetadata } from "../types"
+import { genesisZkeyIndex } from "./constants"
+
+/**
+ * Extract data contained in a logger-generated file containing information extracted from R1CS file read.
+ * @notice useful for extracting metadata circuits contained in the generated file using a logger
```

Comment:
I would make this more clear, something along the lines of: 
"Extract data from a R1CS metadata file generated with a logger and SnarkJS's method r1cs.info()"